tests/image/imageengine.js

```diff
@@ -47,6 +47,16 @@ describe( 'ImageEngine', () => {
 
 				expect( editor.getData() ).to.equal( '<figure class="image"><img src="foo.png"></figure>' );
 			} );
+
+			it( 'should convert with srcset attribute and add sizes attribute', () => {
+				setModelData( document, '<image src="foo.png" alt="alt text" srcset="small.png 148w, big.png 1024w"></image>' );
+
+				expect( editor.getData() ).to.equal(
+					'<figure class="image">' +
+						'<img srcset="small.png 148w, big.png 1024w" sizes="100vw" alt="alt text" src="foo.png">' +
```

Comment:
This order of attributes looks really bad. Especially that below it's completely different. We need to introduce sorting because such tests have a great chance to blow up on different browsers.